app/models/roles_metadata_delegate_regions.rb

```diff
@@ -11,5 +11,8 @@ class RolesMetadataDelegateRegions < ApplicationRecord
 
   has_one :user_role, as: :metadata
   has_one :group, through: :user_role
+  has_one :user, through: :user_role
   has_one :delegate_region, through: :group, source: :metadata, source_type: "GroupsMetadataDelegateRegions"
+
+  scope :senior_delegate, -> { where(status: statuses[:senior_delegate]) }
```

Comment:
I am 97% sure you're getting this scope "for free" because you've declared an `enum` with key `senior_delegate: ...foo...` entry above. In other words, `RolesMetadataDelegateRegions.senior_delegate` should exist and work as intended even without defining this scope.

Reply:
Oh yes, this worked. Thank you.